OrbitCore/Message.h

```diff
@@ -56,7 +56,7 @@ enum MessageType : int16_t {
   Msg_RemoteProcessRequest,
   Msg_RemoteModule,
   Msg_RemoteFunctions,
-  Msg_RemoteProcessList,
+  Msg_RemoteProcessListREMOVED,
```

Comment:
This is renamed instead of removed to not change the value of all the other constants, right? Isn't it better to just remove it (a lot of these values are unused actually) while we don't have compatibility issues?

Reply:
done